bundles/binding/org.openhab.binding.denon/src/main/java/org/openhab/binding/denon/internal/DenonBinding.java

```diff
@@ -137,73 +137,75 @@ protected void removeBindingProvider(DenonBindingProvider bindingProvider) {
      */
     @Override
     public void updated(Dictionary<String, ?> config) throws ConfigurationException {
-        if (config != null) {
-            logger.debug("Denon binding updated");
+        logger.debug("Denon binding updated");
 
-            Enumeration<String> keys = config.keys();
+        if (config == null) {
+            logger.warn("Denon binding has no configuration defined. Check the configuration file.");
```

Comment:
I removed the warning because it didn't exist in the original code (before my changes), so it's effectively no change to not have it there.

However I don't understand why we're claiming it's not needed. If udpated() is passed a null config, the binding will not work. Which means it will fail silently.

Interestingly, I notice that in the current code, if updated is passed a correct configuration, then later receives a null, the binding remains "properly configured", because that flag never gets cleared...